src/parameters/customer_notes.py

```diff
@@ -0,0 +1,11 @@
+import arcpy
+
+def customer_notes_parameter():
+    param = arcpy.Parameter(
+        displayName="Customer Notes",
```

Comment:
I kinda like naming this 'Additional Notes' better rather than 'Customer Notes', what do you think? I'm also totally cool with keeping this as is, I think either makes total sense, but just a thought

Reply:
I agree, I think "additional notes" is a better name for it.